Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -49,6 +98,19 @@ class PreplannedMapModel: ObservableObject, Identifiable {
         }
     }
     
+    /// Sets the model download preplanned offline map job if the job is in progress.
+    private func setDownloadJob() {
```

Comment:
I believe by convention that `set*()` functions should take a parameter. In this case you can probably call it something like:

```suggestion
    private func syncWithJobManager() {
```

Also, this function does much more than what is documented. It also waits for the job to complete and sets the result.